README.md

```diff
@@ -1,2 +1,8 @@
 # hw01
 Repository for homework 01
+#Introduction
+My name is **Lauren Guillot**. I am a student in the *Masters of Arts in the Social Sciences program*. I study political science, specifically [American politics](https://en.wikipedia.org/wiki/American_politics_(political_science) with a focus on public opinion, political behavior, and political psychology.
```

Comment:
One more closing bracket should have been there.
`[American politics](https://en.wikipedia.org/wiki/American_politics_(political_science))`

This is a very common mistake. Each opening bracker should have corresponding closing bracket. 
For example - 
`((a+b + c)` is not valid while `((a+b) + c)` is valid.